mod_reforged/hooks/entity/tactical/enemies/direwolf.nut

```diff
@@ -61,4 +61,33 @@
 			this.m.Skills.add(this.new("scripts/skills/perks/perk_rf_survival_instinct"));
 		}
 	}
+
```

Comment:
The commit message is too long. Please change it to the following i.e. shorter message and move the details to a separate paragraph with a line gap.
```
refactor: adjust loot drops of certain entities

Specifically direwolf, ghoul, hexe, hyena, necromancer, schrat, serpent, spider, unhold, vampire and goblins.
```